Client/Coordinators/EnhancedTrackingProtectionCoordinator.swift

```diff
@@ -2,7 +2,29 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
+import Common
 import Foundation
 
+protocol EnhancedTrackingProtectionCoordinatorDelegate: AnyObject {
+    func didFinishEnhancedTrackingProtection(from coordinator: EnhancedTrackingProtectionCoordinator)
+}
+
 class EnhancedTrackingProtectionCoordinator: BaseCoordinator {
+    private let profile: Profile
+    private let tab: Tab
+    private let enhancedTrackingProtectionMenuVC: EnhancedTrackingProtectionMenuVC
+    weak var parentCoordinator: EnhancedTrackingProtectionCoordinatorDelegate?
+    init(
+        router: Router,
+        profile: Profile = AppContainer.shared.resolve(),
+        tab: Tab = AppContainer.shared.resolve()
```

Comment:
I see the build failed, this is because we can't get the tab this way from `AppContainer`. We should instead pass down the `tabManager` which already exists in BrowserCoordinator. That way it's just a matter of passing the `tabManager` dependency to the ETP coordinator.